content/docs/glossary/liquidity-pool.mdx

```diff
@@ -61,12 +63,256 @@ There are a few possibilities with regard to authorization. The behavior of the
 
 If the issuer of A or B revokes authorization, then the account will automatically withdraw from every liquidity pool containing that asset and those pool share trustlines will be deleted. We say that these pool shares have been _redeemed_. For example, if the account participates in the A-B, A-C, and B-C liquidity pools and the issuer of A revokes authorization then the account will redeem from A-B and A-C but not B-C. For each redeemed pool share trustline, a Claimable Balance will be created for each asset contained in the pool if there is a balance being withdrawn and the redeemer is not the issuer of that asset. The claimant of the Claimable Balance will be the owner of the deleted pool share trustline, and the sponsor of the Claimable Balance will be the sponsor of the deleted pool share trustline. The BalanceID of each Claimable Balance is the SHA-256 hash of the [revokeID](https://github.com/stellar/stellar-core/blob/5bec96c4c9d7080802e80a2e93ddc0bd6bd8a98d/src/xdr/Stellar-transaction.x#L539-L547).
 
+
```

Comment:
nit: not sure how your IDE linter sees it but mine would complain if I add 2 empty lines before a header.